doc/manual.rst

```diff
@@ -5608,11 +5602,11 @@ possible type mismatch error.
 typedesc[T]
 -----------
 
-In many contexts, Nim allows to treat the names of types as regular
-values. These values exists only during the compilation phase, but since
+In many contexts, Nim allows us to treat the names of types as regular
```

Comment:
Maybe just this?
```suggestion
In many contexts, Nim treats the names of types as regular
```

(BTW: Don't take all my suggestions as a single source of truth, I'm not a native English speaker so my suggestions might be incorrect)

Reply:
P.S. Not to worry! I'm not a native English speaker either (although I have some experience with copy editing + I've been speaking and writing English practically all my life, I cannot even tell anymore which language I learned first lol - I guess, my love for programming is the culprit)